lib/src/chart/bar_chart/bar_chart_painter.dart

```diff
@@ -253,8 +265,17 @@ class BarChartPainter extends AxisChartPainter<BarChartData> with TouchHandler<B
                 bottomLeft: borderRadius.bottomLeft,
                 bottomRight: borderRadius.bottomRight);
           }
-
-          _barPaint.color = barRod.color;
+          if (barRod.colors.length == 1) {
+            _barPaint.color = barRod.colors[0];

```

Comment:
Because we are using a shared Paint object, we have to set `_barPaint.shader = null`, if we don't want to draw gradient, otherwise we get this result:

![image](https://user-images.githubusercontent.com/7009300/91425321-3ae09780-e870-11ea-9b62-5955d0ac8bce.png)